social.md

```diff
@@ -76,7 +76,7 @@ The library must implement provider callbacks for all social provider
 directories that are mapped to the specified Stormpath application.  The
 callback URLs for each provider should take the form of:
 
-> `/<stormpath.web.socialProviders.callbackRoot>/<providerId>`
+> `<stormpath.web.socialProviders.[providerId].callbackUri>/<providerId>`
```

Comment:
why not have:

`stormpath.web.social.facebook.uri`

'callbackUri' shouldn't be prefixed with 'callback' - we use 'uri' elsewhere for config, and it is a uri in your application, so should be congruent with other web properties.

Also, why do we need different uris per social provider?

We're moving to a model where a Stormpath URI will be the callback URI (i.e. what you define in Facebook will be something like https://api.stormpath.com/v1/whatever), and, after processing the callback, Stormpath will redirect the user to a uri within the application with a JWT that contains all the information that is necessary.  Multiple URIs not needed.

If this is too much to discuss in a GitHub PR (I assume it might be), then we should probably have a design meeting/Hangout.